app/src/main/java/com/humara/nagar/adapter/ImagePreviewAdapter.kt

```diff
@@ -0,0 +1,50 @@
+package com.humara.nagar.adapter
+
+import android.net.Uri
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.humara.nagar.Logger
+import com.humara.nagar.R
+import com.humara.nagar.utils.Utils
+
+/**
+ * Created by binay on 09,February,2023
+ */
+class ImagePreviewAdapter(
+    private val images: MutableList<Uri>
+) : RecyclerView.Adapter<ImagePreviewAdapter.ViewHolder>() {
+
+    class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+        val imageView: ImageView = itemView.findViewById(R.id.image_preview)
```

Comment:
Should not be using findViewById anywhere. Can take help with WelcomeBannerAdapter class

Reply:
Fixed.